lib/hook.template.raw

```diff
@@ -1,15 +1,53 @@
 #!/usr/bin/env node
 // {{generated_message}}
 
-(function hookEntryPoint() {
+const fs = require('fs')
+const path = require('path')
+
+;(function hookEntryPoint() {
+  const ghooks = getGhooksEntryPoint()
+  checkForGHooks(ghooks)
+  require(ghooks)(__dirname, __filename)
+})()
+
+function getGhooksEntryPoint() {
+  const worktree = getWorkTree()
+  if (worktree) {
+    return path.resolve(__dirname, '../', worktree, 'node_modules', 'ghooks')
+  }
+  return 'ghooks'
+}
+
+function checkForGHooks(ghooks) {
   try {
-    require('ghooks')
+    require(ghooks)
   } catch (e) {
     warnAboutGHooks()
     process.exit(1)
   }
-  require('ghooks')(__dirname, __filename)
-})()
+}
+
+function getWorkTree() {
+  try {
+    return getWorkTreeFromConfig(getConfigFileContent())
+  } catch (e) {
+    return ''
```

Comment:
Yeah, would be more in line with the returned values of `lib/install.js`.

First, I thought of rather changing the `null`s returned into empty strings, but `null` seems more "deterministic" towards the statement "Nope, there's definitely nothing found"...

Thanks for taking the time to review!